test-harness/src/test/java/io/jenkins/plugins/casc/core/NodeMonitorsConfiguratorTest.java

```diff
@@ -0,0 +1,39 @@
+package io.jenkins.plugins.casc.core;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+import hudson.model.ComputerSet;
+import hudson.node_monitors.ArchitectureMonitor;
+import hudson.node_monitors.DiskSpaceMonitor;
+import hudson.node_monitors.ResponseTimeMonitor;
+import io.jenkins.plugins.casc.misc.ConfiguredWithCode;
+import io.jenkins.plugins.casc.misc.JenkinsConfiguredWithCodeRule;
+import jenkins.model.Jenkins;
+import org.junit.Rule;
+import org.junit.Test;
+
+public class NodeMonitorsConfiguratorTest {
+
+    @Rule
+    public JenkinsConfiguredWithCodeRule j = new JenkinsConfiguredWithCodeRule();
```

Comment:
this should use the configuredwithreadmerule instead and should be using your demo so the demo is tested

Reply:
done